master/internal/sproto/task.go

```diff
@@ -138,6 +142,19 @@ type (
 	}
 )
 
+const (
+	// DetRMSupportsProvidingIPAddrs is the name of the env var that defines whether a RM supports
+	// providing IP addrs.
+	DetRMSupportsProvidingIPAddrs = "DET_RM_FEATURE_PROVIDES_IP_ADDRS"
```

Comment:
mostly for greppability
```suggestion
	DetRMFeatureProvidesIPAddrs = "DET_RM_FEATURE_PROVIDES_IP_ADDRS"
```